tests/main/proxy-no-core/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Ensure that the core.proxy.* settings are honored without core
+
+# only needs a test on classic
+systems: [ubuntu-16.04-64, ubuntu-18.04-64]
+
+restore: |
+    iptables -D OUTPUT -m owner --uid-owner "$(id -u test)" -j ACCEPT -p tcp
```

Comment:
Perhaps instead of hand-crafted restore we should use a dump, modify, load approach where here we would just load exactly what was dumped before?